src/ui-config/menu-items/index.tsx

```diff
@@ -83,6 +83,11 @@ const moreMenuItems: MoreMenuItem[] = [
     title: t`Switch to Aave Classic`,
     icon: <AaveClassic />,
   },
+  {
+    link: 'https://global.transak.com',
```

Comment:
If there are multiple `onRampService` providers this will need to change to display a modal or sublinks.

Since this will require additional design, and is complicated due to the component layout, it's fine to leave this for a future PR. Could you open an issue on the main interface repo for this task: 'Change menu links to support multiple on ramps'

Reply:
Done https://github.com/aave/interface/issues/1031